x-pack/test/functional_with_es_ssl/apps/cases/group2/attachment_framework.ts

```diff
@@ -325,6 +331,88 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         });
       });
     });
+
+    describe('Lens visualization as persistable attachment', () => {
+      const myDashboardName = `My-dashboard-${uuidv4()}`;
```

Comment:
Just curiosity but why a `uuid` vs just calling it `My-dashboard`?

Reply:
if someone creates a dashboard with same name (My-dashboard) in this file, the test will fail because that dashboard already exists. But with uuid, it will always be unique.

Reply:
> But with uuid, it will always be unique.

yy ok, I just thought it was overkill to use `uuid` instead of using some creative name like `My-dashboard-foobar` 😆 

But nevermind,  no need to do anything 👍